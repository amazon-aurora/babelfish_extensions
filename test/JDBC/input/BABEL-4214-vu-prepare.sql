CREATE VIEW babel_4214_datetime_to_int_view
AS
SELECT 1;
GO