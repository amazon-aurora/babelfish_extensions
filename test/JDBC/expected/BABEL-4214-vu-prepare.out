CREATE VIEW babel_4214_datetime_to_int_view
AS
SELECT CAST(CAST('1900-01-10 17:09:58.003' AS DATETIME) AS INT);
GO

CREATE PROC babel_4214_datetime_to_int_proc
AS
SELECT CAST(CAST('1900-01-10 17:09:58.003' AS DATETIME) AS INT);
GO

CREATE FUNCTION babel_4214_datetime_to_int_function()
RETURNS INT AS
BEGIN
    RETURN (SELECT CAST(CAST('1900-01-10 17:09:58.003' AS DATETIME) AS INT))
END
GO

CREATE VIEW babel_4214_smalldatetime_to_int_view
AS
SELECT CAST(CAST('1900-01-10 17:09:58.003' AS SMALLDATETIME) AS INT);
GO

CREATE PROC babel_4214_smalldatetime_to_int_proc
AS
SELECT CAST(CAST('1900-01-10 17:09:58.003' AS SMALLDATETIME) AS INT);
GO

CREATE FUNCTION babel_4214_smalldatetime_to_int_function()
RETURNS INT AS
BEGIN
    RETURN (SELECT CAST(CAST('1900-01-10 17:09:58.003' AS SMALLDATETIME) AS INT))
END
GO
